cmd/web/main.go

```diff
@@ -75,6 +75,10 @@ func main() {
 	api.DELETE("/clusters", server.ClusterDeleteHandler)
 	api.GET("/resourcegroups/:accountID", server.ResourceGroupHandler)
 	api.GET("/iam/accessGroups/:accountID", server.AccessGroupsHandler)
+	api.PUT("/groups/:accessGroupID/members", server.AddMemberHandler)
+	api.POST("/accounts/:accountID/users", server.InviteUserHandler)
+	api.POST("/policies", server.CreatePolicyHandler)
```

Comment:
should policy be under `/iam/` since its part of iam?

Reply:
```suggestion
      api.POST("/iam/policies", server.CreatePolicyHandler)
```

Reply:
I agree. I can fix this.

Reply:
They could all fall under IAM.